docs/tutorials/setting-up-an-nfs-fileShare.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Setting Up an NFS Fileshare
+subject: Networking
+author: Anton Vanhoucke, Ralph Hempel
+---
+
+* Table of Contents
+{:toc}
+
+## Why Do It?
```

Comment:
I think that there should be a notice at the top saying that this is generally for advanced users. We don't want people who aren't prepared to do a some manual configuring getting stuck here.

You can use an info-style alert for this:
```
{% include icon.html type="info" %}
This is your info text
{: .alert .alert-info}
```